packages/integration-karma/test/rendering/inheritance/index.spec.js

```diff
@@ -0,0 +1,24 @@
+import { createElement } from 'lwc';
+import Foo from 'x/foo';
+import Bar from 'x/bar';
+
+describe('template inheritance', () => {
+    it('should support implicit definition of new template', function() {
+        const elm = createElement('x-foo', { is: Foo });
+        document.body.appendChild(elm);
+
+        return Promise.resolve().then(() => {
```

Comment:
The assertion can be done synchronously. 
I tried this locally without the `return Promise.resolve().then(() => {});` and the test consistently pass.